projects/plugins/jetpack/extensions/plugins/seo/counted-textarea.js

```diff
@@ -12,8 +12,12 @@ export const CountedTextArea = ( { suggestedLimit, value, label, ...inputProps }
 			  )
 			: sprintf(
 					/* translators: Placeholder is a number of characters in a sentence. */
-					__( "It's recommended to use less than %d characters in this field.", 'jetpack' ),
-					suggestedLimit
+					__(
+						"It's recommended to use less than %1$d characters in this field. (%2$d used)",
```

Comment:
I would take that opportunity to move away from the double quotes and use a real apostrophe, maybe?

```suggestion
						'It’s recommended to use less than %1$d characters in this field. (%2$d used)',
```